extension-androidauto/src/main/java/com/mapbox/maps/extension/androidauto/MapboxCarMapEx.kt

```diff
@@ -0,0 +1,36 @@
+@file:kotlin.jvm.JvmName("MapboxCarMapEx")
+
+package com.mapbox.maps.extension.androidauto
+
+import androidx.car.app.CarContext
+import androidx.car.app.Screen
+import androidx.car.app.Session
+import com.mapbox.maps.MapInitOptions
+import com.mapbox.maps.MapboxExperimental
+
+/**
+ * Shorthand extension for installing the map onto a [Session].
+ */
+@OptIn(MapboxExperimental::class)
```

Comment:
nit: as this extension function is also a public API surface, shall we just use `@MapboxExperimental`?
```suggestion
@MapboxExperimental
```